tiledb/libtiledb.pyx

```diff
@@ -2620,10 +2620,10 @@ cdef class Dim(object):
                     # this means to use the full extent of the type
                     if dtype.kind == 'M':
                         date_unit = np.datetime_data(dtype)[0]
-                        dtype_min = np.datetime64(dtype_min + 1, date_unit)
+                        dtype_min = np.datetime64(dtype_min, date_unit)
```

Comment:
Isn't this expression what `dtype_range` does to compute `dtype_min`? I think it's not needed anymore.

Reply:
✔️